scripts/compile_flattened.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -ex
+
+PRAGMA="// SPDX-License-Identifier:MIT\npragma solidity ^0.6.10;\npragma experimental ABIEncoderV2;"
+CONTRACTS=("RelayHub" "StakeManager" "Penalizer" "Forwarder" "TestPaymasterEverythingAccepted")
+
+# ok, I have no idea how to do it right; if it's working it ain't stupid though
+PATHS=("./" "./" "./" "./forwarder/" "./test/")
+
+INFOLDER=./contracts
+TMPFOLDER=./build/flattened/
+OUTFOLDER=./src/cli/compiled
+
+mkdir -p $TMPFOLDER
+
+END=${#CONTRACTS[@]}-1
+for ((i=0;i<=END;i++)); do
+  c=${CONTRACTS[$i]}
+  c_flat=$TMPFOLDER"${c}"_flat.sol
+  echo -e $PRAGMA > $c_flat
+  truffle-flattener $INFOLDER/${PATHS[$i]}$c.sol | grep -v pragma | grep -v SPDX >> $c_flat
```

Comment:
if flattener is that broken that it doesn't handle the source code well, then we should't depend on it.
(e.g. we KNOW the code changes because of the use of pragma... so why do we test one thing, and deploy another?)